syntax = "proto3";

import "protos/models/security/identifier_type.proto";

package security;

option java_multiple_files = true;
option java_package = "common.model.protos";
option java_outer_classname = "IdentifierProtos";

message IdentifierProto {
  string object_class = 1;
  string version = 2;

  //Primary Key
  string identifier_value = 5;
  IdentifierTypeProto identifier_type = 6;
}
